test/parallel/test-buffer-includes.js

```diff
@@ -79,44 +79,74 @@ assert(!b.includes(Buffer.from('f'), 6));
 assert(!Buffer.from('ff').includes(Buffer.from('f'), 1, 'ucs2'));
 
 // test hex encoding
-assert(
-    Buffer.from(b.toString('hex'), 'hex')
-    .includes('64', 0, 'hex'));
-assert(
-    Buffer.from(b.toString('hex'), 'hex')
-    .includes(Buffer.from('64', 'hex'), 0, 'hex'));
+assert.strictEqual(
+  Buffer
+    .from(b.toString('hex'), 'hex')
+    .includes('64', 0, 'hex'),
+  true
+);
+assert.strictEqual(
+  Buffer
+    .from(b.toString('hex'), 'hex')
+    .includes(Buffer.from('64', 'hex'), 0, 'hex'),
+  true
+);
```

Comment:
@Slayer95 This was a bit outside the scope of what is described in the commit message, so (as @targos suggested previously), I should update the commit message. This is an improvement (in my opinion, anyway) because the previous test passed as long as `buf.includes()` returned any truthy value. But, in fact, `buf.includes()` is broken if it returns some other value than `true`. So this makes the test more strict.

However, in the process of making the test better, we've made the assertion less readable because now there is a second argument and the previous layout messes with the second argument.

If you do this, then it's easy to miss the second argument entirely:

    assert.strictEqual(
      Buffer.from(b.toString('hex'), 'hex')
     .includes(Buffer.from('64', 'hex'), 0, 'hex'), true
    );

This is better, but it's not immediately clear that `true` is the second argument and not the third:

    assert.strictEqual(
      Buffer.from(b.toString('hex'), 'hex')
      .includes(Buffer.from('64', 'hex'), 0, 'hex'),
      true
    );

This next one is more lines to take in at once than might be ideal, but once you see the pattern, it's easy to scan the subsequent occurrences of the pattern. The same is not true for the above. They are more difficult to scan, especially if the number of chained properties in the first argument varies or if the number of arguments sent to the function varies. So, in my opinion, it is the more readable layout:

    assert.strictEqual(
      Buffer
        .from(b.toString('hex'), 'hex')
        .includes(Buffer.from('64', 'hex'), 0, 'hex'),
      true
    );